lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2627,6 +2643,41 @@ def _add_to_dict(metadata):
         return d
 
 
+def loggly_time_string(dt):
+    formatted_dt = dt.strftime("%Y-%m-%dT%H:%M:%S")
+    microseconds = str(round(dt.microsecond * (10.0**-5), 3))
+    return urllib.quote_plus(formatted_dt + microseconds + "Z")
+
+
+def get_loggly_query_string(lbry_id):
+    decoded_id = base58.b58encode(lbry_id)
```

Comment:
Ah - crap - this is a reminder for me to fix https://github.com/lbryio/lbry/blob/2afc4c25a295cee21e84e739171f93405b8445b5/lbrynet/lbrynet_daemon/Daemon.py#L1239 to use base58, not hexlify.